app/views/users/webauthn_setup/new.html.erb

```diff
@@ -30,20 +30,24 @@
   <%= hidden_field_tag :client_data_json, '', id: 'client_data_json' %>
   <%= hidden_field_tag :authenticator_data_value, '', id: 'authenticator_data_value' %>
   <%= hidden_field_tag :transports, '', id: 'transports' %>
+
   <%= hidden_field_tag :platform_authenticator, @platform_authenticator, id: 'platform_authenticator' %>
-  <%= render ValidatedFieldComponent.new(
-        form: f,
-        name: :name,
-        required: true,
-        label: @presenter.nickname_label,
-        hint: @presenter.device_nickname_hint,
-        input_html: {
-          id: 'nickname',
-          class: 'font-family-mono',
-          size: 16,
-          maxlength: 20,
-        },
-      ) %>
+  <% if !@platform_authenticator %>
```

Comment:
Can we add some test coverage to the view spec to assert that the input is only present based on the `@platform_authenticator` value?

Reply:
Yes. Added that.